build/templates/reportTemplate.html.j2

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <link href="./static/style.css" rel="stylesheet">
+      <link href="./static/bootstrap-theme.min.css" rel="stylesheet">
```

Comment:
NO RELATIVE LINKS

Reply:
all caps because we discussed this. A web server is going to serve this path. It should be an absolute route to the webserver root `/static`

Reply:
removed '.'s